mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
+CC_x86_64-unknown-linux-uclibc=$(CFG_UCLIBC_ROOT)/bin/uclibc-gcc
+CXX_x86_64-unknown-linux-uclibc=notaprogram
+CPP_x86_64-unknown-linux-uclibc=$(CFG_UCLIBC_ROOT)/bin/uclibc-gcc -E
+AR_x86_64-unknown-linux-uclibc=$(AR)
+CFG_INSTALL_ONLY_RLIB_x86_64-unknown-linux-uclibc = 1
+CFG_LIB_NAME_x86_64-unknown-linux-uclibc=lib$(1).so
+CFG_STATIC_LIB_NAME_x86_64-unknown-linux-uclibc=lib$(1).a
+CFG_LIB_GLOB_x86_64-unknown-linux-uclibc=lib$(1)-*.so
+CFG_JEMALLOC_CFLAGS_x86_64-unknown-linux-uclibc := -m64
+CFG_GCCISH_CFLAGS_x86_64-unknown-linux-uclibc := -Wall -Werror -g -fPIC -m64
+CFG_GCCISH_CXXFLAGS_x86_64-unknown-linux-uclibc :=
+CFG_GCCISH_LINK_FLAGS_x86_64-unknown-linux-uclibc :=
+CFG_GCCISH_DEF_FLAG_x86_64-unknown-linux-uclibc :=
+CFG_LLC_FLAGS_x86_64-unknown-linux-uclibc :=
+CFG_INSTALL_NAME_x86_64-unknown-linux-uclibc =
+CFG_EXE_SUFFIX_x86_64-unknown-linux-uclibc =
+CFG_WINDOWSY_x86_64-unknown-linux-uclibc :=
+CFG_UNIXY_x86_64-unknown-linux-uclibc := 1
+CFG_LDPATH_x86_64-unknown-linux-uclibc :=
+CFG_RUN_x86_64-unknown-linux-uclibc=$(2)
+CFG_RUN_TARG_x86_64-unknown-linux-uclibc=$(call CFG_RUN_x86_64-unknown-linux-uclibc,,$(2))
+CFG_GNU_TRIPLE_x86_64-unknown-linux-uclibc := x86_64-unknown-linux-uclibc
+CFG_THIRD_PARTY_OBJECTS_x86_64-unknown-linux-uclibc := crt1.o crti.o crtn.o
```

Comment:
You don't need to list `crt*.o`, `libc.a` or `libunwind.a` here because this target is not building statically linked binaries like the musl target.